orchagent/swssnet.h

```diff
@@ -76,6 +77,53 @@ inline static sai_ip_prefix_t& copy(sai_ip_prefix_t& dst, const IpAddress& src)
     return dst;
 }
 
+static int getPrefixLenFromAddr(const uint8_t *addr, int len)
+{
+    int i = 0;
+    uint8_t non_zero = 0xFF;
+    for (i = len - 1; i >=0; i--)
+    {
+        if (addr[i] != 0)
+        {
+            non_zero = addr[i];
+            break;
+        }
+    }
+
+    if (non_zero == 0xFF)
+    {
+        return (i + 1) * 8;
+    }
+    else
+    {
+        int j = 2;
+        while(((non_zero >> j) & 0x1) == 0)
+        {
+            ++j;
+        }
+        return (i + 1) * 8 - (j + 1);
+    }
+    
+}
+
+inline static IpPrefix convert(const sai_ip_prefix_t& src)
```

Comment:
I don't think we should be having such a utility to convert back. Please revisit the use-case.

Reply:
This change is for your previous comments: https://github.com/Azure/sonic-swss/pull/2094#discussion_r834790424 
In order to fix this, need to convert from sai_ip_prefix to IpPrefix.

Reply:
ok, then suggest rename to `getIpPrefixFromSaiPrefix`

Reply:
I don't see this addressed

Reply:
Fixed